test/built-ins/Math/pow/int32_min-exponent.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2018 Mozilla Corporation. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-applying-the-exp-operator
+description: >
+    Using -(2**31) as exponent with Math.pow should behave as expected.
+---*/
+
+const INT32_MIN = -2147483648;
+
+assert.sameValue(Math.pow(2, INT_MIN), +0.0,
```

Comment:
nit: (twice in this file, and twice in the other file) `INT32_MIN`, not `INT_MIN`.

Reply:
Unfortunately, this can't be asserted in Test262. The result is implementation-dependent so the specs won't recommend over edge cases of the exponent value.
 
```
12.6.4 Applying the ** Operator

Returns an implementation-dependent approximation of the result of raising base to the power exponent.

...
```

I'd like to agree with obvious cases, e.g. `2**3`, `2**100`, but going something far from this is something we need to rely on the implementation.

My best suggestion is to propose some basic spec expectations for integer operations limited by IEEE 754 edge values.